input/docs/integrations/editors/rider/run-configurations.md

```diff
@@ -1,4 +1,4 @@
-Order: 30
+Order: 20
```

Comment:
Why changing order? IntelliSense page has already order 20 (and should be before this page to be consistent with other integration docs)

Reply:
Ah. Because I'm bad at merging things.

Reply:
@pascalberger I fixed that.